src/include/daos/tests_lib.h

```diff
@@ -166,11 +166,15 @@ struct dts_context {
 	/** OUTPUT END */
 };
 
+#define MAX_TEST_TARGETS_PER_DEVICE 10
```

Comment:
If the test server has 16 targets and 1 device, it'll be 16 targets per device. You'd make it bigger, like 48 ? (48 is used in BIO, see BIO_XS_CNT_MAX).

Reply:
will update